### PR TITLE
feat(zk_toolbox): Add zk_supervisor run unit tests command

### PR DESCRIPTION
**Pull request by matias-gonz**
## What ❔
Add zk_supervisor run unit tests command

### CODE REVIEW

File: zk_toolbox/crates/zk_supervisor/src/commands/test/rust.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+use common::{cmd::Cmd, logger};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+use super::args::rust::RustArgs;
+use crate::messages::{
+    MSG_CARGO_NEXTEST_MISSING_ERR, MSG_UNIT_TESTS_RUN_SUCCESS, MSG_USING_CARGO_NEXTEST,
+};
+
+pub fn run(shell: &Shell, args: RustArgs) -> anyhow::Result<()> {
```
Comment (Deniallugo) [inline zk_toolbox/crates/zk_supervisor/src/commands/test/rust.rs:10]: you have to setup the database as well. some of our tests are using the test-database. 
Please make sure, that you are not removing the ~production~ database

File: zk_toolbox/crates/zk_supervisor/src/commands/test/rust.rs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            .run_with_output()?
+            .stdout,
+    )?;
+    Ok(out.contains("cargo-nextest"))
+}
+
+fn wait_for_dal(shell: &Shell, dal: &Dal) -> anyhow::Result<()> {
+    let url = dal.url.as_str();
+    for _ in 0..10 {
+        if let Ok(out) = Cmd::new(cmd!(shell, "pg_isready -d {url}")).run_with_output() {
```
Comment (Deniallugo) [inline zk_toolbox/crates/zk_supervisor/src/commands/test/rust.rs:77]: we have `wait_for_db` in `common/db`

File: zk_toolbox/crates/zk_supervisor/src/dals.rs [hunk 3]
```diff
@@ -41,6 +43,14 @@ pub fn get_dals(shell: &Shell, selected_dals: &SelectedDals) -> anyhow::Result<V
     Ok(dals)
 }
 
+pub fn get_test_dals(shell: &Shell) -> anyhow::Result<Vec<Dal>> {
+    let mut dals = vec![];
+    //dals.push(get_test_prover_dal(shell)?);
```
Comment (Deniallugo) [inline zk_toolbox/crates/zk_supervisor/src/dals.rs:48]: why?

Comment (matias-gonz) [reply zk_toolbox/crates/zk_supervisor/src/dals.rs:48]: Forgot to re-add it!

**Review by Deniallugo — APPROVED** [review]

**Pull request merged**